skyportal/facility_apis/tarot.py

```diff
@@ -539,11 +603,34 @@ def delete(request, session, **kwargs):
     form_json_schema_altdata = {
         "type": "object",
         "properties": {
-            "token": {
+            "request_id": {
```

Comment:
Can you make sure the docs/ with the followup telescope parameters and where they come from are updated too?